### ISSUE TITLE
New and improved Skor is now part of Hasura GraphQL Engine

### ISSUE THREAD

**Issue by shahidhk** [hasura/skor#23]
## New and improved version of Skor is now part of [Hasura GraphQL Engine](https://github.com/hasura/graphql-engine)

A few months ago, we built the open source [GraphQL Engine](https://github.com/hasura/graphql-engine) that gives you instant GraphQL APIs over any Postgres database. We have added all of Skor's existing features and even more to make it production ready:

 1) Reliable: We capture every relevant action on the database as an event, even when Hasura is down! The events are delivered to your webhook as soon as possible with an atleast-once guarantee.

2) Scalable: What more, it even scales horizontally. If you are processing millions of events, just add more instances of GraphQL engine.

3) Use with Serverless: If you are using Skor, then avoid the pain of managing your webhook by moving to Serverless infrastructure. Check out these blog posts to get started

**Use [Hasura GraphQL Engine](https://github.com/hasura/graphql-engine) for production use cases**

**Comment by coco98** [comment]
@shahidhk Would recommend these change.
In scalable, We can just say: Event delivery scales horizontally. If you want to process more events faster, just increase the number of instances of Hasura. (I'm in favour of gradually dropping graphql engine from our terminology)

In point 3:
```
3. Use with serverless: Webhook triggers on postgres events work especially great with serverless. We've put together a bunch of useful getting started's here.
```

I think we should add a point around there being a UI to configure event-triggers easily too. We have a nice GIF. You can get it from my blogpost.

I don't think you need to have the last line "Use HGE for production....."

We can also serve a deprecation notice saying that work on skor will now continue has a part of HGE.

**Comment by sachaarbonel** [comment]
> ## New and improved version of Skor is now part of [Hasura GraphQL Engine](https://github.com/hasura/graphql-engine)
> 
> A few months ago, we built the open source [GraphQL Engine](https://github.com/hasura/graphql-engine) that gives you instant GraphQL APIs over any Postgres database. We have added all of Skor's existing features and even more to make it production ready:
> 
>     1. Reliable: We capture every relevant action on the database as an event, even when Hasura is down! The events are delivered to your webhook as soon as possible with an atleast-once guarantee.
> 
>     2. Scalable: What more, it even scales horizontally. If you are processing millions of events, just add more instances of GraphQL engine.
> 
>     3. Use with Serverless: If you are using Skor, then avoid the pain of managing your webhook by moving to Serverless infrastructure. Check out these blog posts to get started
> 
> 
> **Use [Hasura GraphQL Engine](https://github.com/hasura/graphql-engine) for production use cases**


Does this mean that when you have N hasura instances running for scalability and for example an event-trigger to push notification to one subscriber, this subscriber will receive only 1 notification and not N notifications?

**Comment by tirumaraiselvan** [comment]
@Sach97 Yes, absolutely. The one event will be picked by any one of the N Hasura instances and sent to the subscriber.

**Comment by sachaarbonel** [comment]
Oh that's nice thank's, that is exactly what I was searching for !
